html/en/index.html

```diff
@@ -129,9 +135,9 @@ <h5 class="modal-title">
     <script src="../../../website/src/script/index.ts"></script>
     <script>
         app.countries = "";
-        app.style = "mapbox://styles/mapbox/dark-v10";
+        app.style = undefined;
```

Comment:
You can even remove the line !

```suggestion
```

Reply:
Yes that's possible, but now it's clear that the map style can be changed.

Reply:
I'll add instructions in the `README.md` file.